cli/tests/unit/signal_test.ts

```diff
@@ -96,6 +96,63 @@ unitTest(
   },
 );
 
+unitTest(
+  { ignore: Deno.build.os !== "windows", perms: { run: true, net: true } },
+  async function signalWindowsStreamTest(): Promise<void> {
+    // This prevents the program from exiting.
+    const t = setInterval(() => {}, 1000);
+    const resolvable = deferred();
+    let c = 0;
+    const sig = Deno.signal(Deno.Signal.SIGINT);
+    /*setTimeout(async () => {
+      await defer(20);
+      for (const _ of Array(3)) {
+        // Sends SIGINT 3 times.
+        Deno.kill(Deno.pid, Deno.Signal.SIGINT);
+        await defer(20);
+      }
+      sig.dispose();
+      resolvable.resolve();
+    });*/
```

Comment:
This commented out original code confuses me, running unit_test_runner.ts in clion terminal or using deno to run this code alone in windows terminal are all expected behaviors.
But running unit_test_runner.ts in windows terminal (or cmd.exe), this code will only send a signal signal (kill is only executed once?), even if the defer time is increased or sig.dispose is commented out, the same will be obtained. .
I had to use setTimeout(() => Deno.kill(Deno.pid, Deno.Signal.SIGINT), 20) to achieve the desired effect.